docs/extending/rich_text_internals.md

```diff
@@ -106,14 +114,18 @@ You can create custom rewrite handlers to support your own new `linktype` and `e
 
     .. method:: get_instance(attrs)
 
-        Optional. The static or classmethod ``get_instance`` method also only applies to those handlers that are used to render content related to Django models. This method is expected to take a dictionary of attributes from a database rich text ``<a>`` tag (``<embed>`` for ``EmbedHandler``) and use it to return the specific Django model instance being referred to.
+        Optional. The classmethod ``get_instance`` method also only applies to those handlers that are used to render content related to Django models. This method is expected to take a dictionary of attributes from a database rich text ``<a>`` tag (``<embed>`` for ``EmbedHandler``) and use it to return the specific Django model instance being referred to.
 
         For example, ``PageLinkHandler.get_instance`` might receive ``{'id': 123}`` and return the instance of the Wagtail ``Page`` class with ID 123.
 
         If left undefined, a default implementation of this method will query the ``id`` model field on the class returned by ``get_model`` using the provided ``id`` attribute; this can be overridden in your own handlers should you want to use some other model field.
+
+    .. method:: get_many(attrs_list)
+
+        Optional. The classmethod ``get_many`` method works similarly to ``get_instance`` but instead takes a list of attribute dictionaries and returns a list of Django model instances.
```

Comment:
I think it's worth documenting what happens when an instance is not found - namely, returning `None` for that item in the list. (And I suppose for completeness the documentation for `get_instance` should mention that it throws an `ObjectDoesNotExist` exception in that situation.)

Reply:
Thanks for this suggestion -- the existing `MediaEmbedHandler` [raises `EmbedUnsupportedProviderException`](https://github.com/wagtail/wagtail/blob/f30c65138e1cb2162134f15976aec4b1b383752c/wagtail/embeds/embeds.py#L32) if the embed URL isn't supported, not `ObjectDoesNotExist`.

I used a slightly less specific wording that tries to capture this case:

> This method should raise an exception if the provided attributes cannot be used to retrieve a Django model instance, for example if the provided `id` attribute is invalid.